src/Michał/Fsd.Michal.Cs.Ex2/FarmGenerator/FarmGenerator/ShowData.cs

```diff
@@ -0,0 +1,22 @@
+﻿using FarmGenerator.Models;
+using System;
+using System.Collections.Generic;
+
+namespace FarmGenerator.Interfaces
+{
+    class ShowData
```

Comment:
ShowData to nazwa dobra dla metody (Czasownik). Klasy nazywaj raczej rzeczownikami